src/admin.rs

```diff
@@ -675,14 +675,16 @@ mod tests {
                     network: "defaultnw".to_string(),
                     address: [127, 0, 0, 10].to_vec(),
                 })),
-                port: 15008,
+                hbone_mtls_port: 15008,
+                hbone_single_tls_port: Some(15006),
```

Comment:
```suggestion
                hbone_single_tls_port: Some(15006),
```

Is it smart to recontextualize this (existing) port we use for plaintext?

I get that we should treat single-tls as untrusted, but should we dual-purpose the port already used for no-tls?

Should we just pick a new port?

Reply:
My understanding is what the waypoint didn't have a plaintext port

Reply:
it doesn't, but ztunnel does, and already uses 15006 for this purpose, so it's a bit confusing to repurpose that same port for this single-tls hairpin functionality, versus picking a new dedicated port for this.

(there might be an argument here that I'm missing tho)

Reply:
Hmm not a bad idea. 15009?

Reply:
> Hmm not a bad idea. 15009?

Or [15003 to give us an "empty slot" on either side for future use](https://github.com/istio/ztunnel/blob/master/ARCHITECTURE.md), but yeah whatever you prefer/others have opinions on.